ui/StatusQ/src/StatusQ/Components/StatusRoundedMedia.qml

```diff
@@ -80,6 +94,11 @@ StatusRoundedComponent {
         return StatusRoundedMedia.MediaType.Unknown
     }
 
+    signal imageClicked(var image, bool plain)
```

Comment:
sounds like we could have 2 single, generic signals (mediaClicked, openMediaContextMenu) if we add the mediaType to the parameters.